core/src/main/java/jenkins/model/ParameterizedJobMixIn.java

```diff
@@ -278,7 +278,8 @@ public static final CauseAction getBuildCause(ParameterizedJob job, StaplerReque
      * Uses {@link #BUILD_NOW_TEXT}.
      */
     public final String getBuildNowText() {
-        return isParameterized() ? Messages.ParameterizedJobMixIn_build_with_parameters() : AlternativeUiTextProvider.get(BUILD_NOW_TEXT, asJob(), Messages.ParameterizedJobMixIn_build_now());
+        return isParameterized() ? AlternativeUiTextProvider.get(BUILD_NOW_TEXT, asJob(), Messages.ParameterizedJobMixIn_build_with_parameters())
```

Comment:
Shouldn't this rather have a new variant `BUILD_WITH_PARAMETERS_TEXT`? The difference IIUC is that "Build Now" indicates an immediate action, while the other does not (like when in many applications some actions have a trailing `…` and others don't).